static/css/addons-form.css

```diff
@@ -219,3 +219,11 @@
 #addon-config-settings .button-submit:active {
   background-color: #658196;
 }
+
```

Comment:
This should go in `settings.css`, along with the other styles for this element. Also, the link should always be white. To make that happen, change this block in `settings.css`:

```css
.addon-settings-license,
.addon-settings-author {
  font-size: 1.4rem;
  font-style: italic;
  color: #ccc;
}

.addon-settings-license {
  padding-left: 0.5rem;
}

.addon-settings-license a:link,
.addon-settings-license a:visited,
.addon-settings-license a:hover,
.addon-settings-license a:active,
.addon-settings-author a:link,
.addon-settings-author a:visited,
.addon-settings-author a:hover,
.addon-settings-author a:active {
  color: #fff;
}
```

to:

```css
.addon-settings-license,
.addon-settings-author {
  font-size: 1.4rem;
  font-style: italic;
}

.addon-settings-author {
  color: #ccc;
}

.addon-settings-author a:link,
.addon-settings-author a:visited,
.addon-settings-author a:hover,
.addon-settings-author a:active {
  color: #fff;
}

.addon-settings-license {
  padding-left: 0.5rem;
  cursor: pointer;
  color: #fff;
}
```

Reply:
done